Ui improvements

ayoungdrew opened the pull request.

ayoungdrew committed:
Add product detail view

- New handlebars template for individual product detail view
- Fixed bug that appeared upon product detail view - set all Products
initiates on page load now, not just on sign in

Andrew, Ian

ayoungdrew committed:
Breadcrumb links now work for product detail view

- Adds breadcrumb anchor tags
- New add handlers to show all products or just the category of Products
based on which breadcrumb is clicked.

Andrew

ismurray reviewed (changes requested):
Could you make it also hide the product view on signout? Looks like there's some visual artifacts happening with hide/show

ayoungdrew committed:
Fixed minor product detail view bug + button css

- Product detail view was lingering on sign-out or return to homepage
- Slight edit to button styling

ayoungdrew committed:
Fixes bug that showed no past purchases date

- Was a minor typo in cartPase.js
- Minor padding styling in cart modal
- Removed some modal artifacts that were unneeded

Andrew, Ian
